### ISSUE TITLE
Currency rates not being properly tracked

### ISSUE THREAD

**Issue by Biggoron144** [briansd9/exile-diary#40]
Currency value for stash items does not appear to be tracked correctly.

https://pastebin.com/WpNdZ7mD for the log txt

**Comment by briansd9** [comment]
You're right, there was a problem getting item prices from poe.ninja... can you check if this URL works from your end? https://poe.ninja/api/data/itemoverview?type=BaseType&league=Metamorph

If so, restarting the app should allow the prices to be retrieved properly.

Thanks for the report, will make sure this is better handled in the next release.

**Comment by Biggoron144** [comment]
That link appears to be working. First few lines:

{"lines":[{"id":32013,"name":"Two-Toned Boots","icon":"https://web.poecdn.com/image/Art/2DItems/Armours/Boots/TwoTonedBoots3B.png?scale=1&w=2&h=2","mapTier":0,"levelRequired":85,"baseType":"Two-Toned Boots","stackSize":0,"variant":"Redeemer","prophecyText":null,"artFilename":null,"links":0,"itemClass":2,"sparkline":{"data":[],"totalChange":0.0},"lowConfidenceSparkline":{"data":[null,null,null,null,0.0,-66.35,2814.74],"totalChange":2814.74},"implicitModifiers":[],"explicitModifiers":[],"flavourText":"","corrupted":false,"gemLevel":0,"gemQuality":0,"itemType":"Boots","chaosValue":9226.89,"exaltedValue":84.43,"count":1,"detailsId":"two-toned-boots-85-redeemer"},{"id":32347,"name":"Marble Amulet","icon":"https://web.poecdn.com/image/Art/2DItems/Amulets/MarbleAmulet.png?scale=1&w=1&h=1","mapTier":0,"levelRequired":85,"baseType":"Marble Amulet","stackSize":0,"variant":"Crusader","prophecyText":null,"artFilename":null,"links":0,"itemClass":2,"sparkline":{"data":[],"totalChange":0.0},"lowConfidenceSparkline":{"data":[null,null,null,null,null,0.0,11.75],"totalChange":11.75},"implicitModifiers":[],"explicitModifiers":[],"flavourText":"","corrupted":false,"gemLevel":0,"gemQuality":0,"itemType":"Amulet","chaosValue":9226.89,"exaltedValue":84.43,"count":1,"detailsId":"marble-amulet-85-crusader"},{"id":33758,"name":"Ebony Tower Shield","icon":"https://web.poecdn.com/image/Art/2DItems/Armours/Shields/ShieldStr4.png?scale=1&w=2&h=4","mapTier":0,"levelRequired":84,"baseType":"Ebony Tower Shield","stackSize":0,"variant":"Hunter","prophecyText":null,"artFilename":null,"links":0,"itemClass":2,"sparkline":{"data":[],"totalChange":0.0},"lowConfidenceSparkline":{"data":[null,null,null,null,null,null,0.0],"totalChange":0.0},"implicitModifiers":[],"explicitModifiers":

**Comment by Biggoron144** [comment]
Seems to be working in game now.
